src/main/java/ua/kpi/iasa/IASA_Organiser/service/EventBuilder.java

```diff
@@ -0,0 +1,85 @@
+package ua.kpi.iasa.IASA_Organiser.service;
+
+import ua.kpi.iasa.IASA_Organiser.model.*;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.UUID;
+
+public class EventBuilder implements Builder{
```

Comment:
Why do we need that class?
@makrushinoff

Reply:
We just NEED this class @MLGMAG